CMSgov/design-system
# [WNMGDS-1885] Remove deprecated i18n props

## Summary

[WNMGDS-1885](https://jira.cms.gov/browse/WNMGDS-1885)

- Removed deprecated i18n props, documentation, warnings, etc..
- Updated several tests to give the correct output

## How to test

1. `yarn clean && yarn install && yarn build`
2. `yarn test`
3. `yarn storybook` - check out `MonthPicker`, `UsaBanner`, HC's `Logo` and HC's `Header` primarily

## Checklist


### If this is a change to documentation:
